templates/fullstack/server/src/schema/schema.graphql

```diff
@@ -1,29 +1,87 @@
 ## NOTE: This schema was generated by Graphback and should not be changed manually
 
+"""Exposes a URL that specifies the behaviour of this scalar."""
```

Comment:
Our approach is schema/code/mapping first at the same time. 
Developers describe their datarequirements using models (that are graphql schema and getting fully featured schema in results.

Workflows are very flexible - do not like graphback resolver layer - generate just schema (using plugins) - Do not like client side queries etc.